consensus/checks.go

```diff
@@ -141,24 +144,6 @@ func (consensus *Consensus) onPreparedSanityChecks(
 			Msg("[OnPrepared] BlockHash not match")
 		return false
 	}
-	if consensus.current.Mode() == Normal {
```

Comment:
why is this removed?

Reply:
It is not removed but instead the logic is rewritten. Previously, if the node is in syncing or listening mode, the block will be always added to FBFT log without block verification. This might cause consensus memory have some malicious block. Thus, the block verification logic is also applied to all blocks in FBFTLogs and added `verifiedBlocks` in FBFTLog. The block verification logics are:

1. tryCatchup (consensus/consensus_v2.go:495)
2. `LastMileBlockIterator.Next` (consensus/consensus_v2.go:446)
3.  `onPrepared` (consensus/validator.go:204)